src/app/Library/CrudPanel/Traits/Access.php

```diff
@@ -38,6 +39,13 @@ public function hasAccess(string $operation, $entry = null): bool
     {
         $condition = $this->get($operation.'.access');
 
+        // this shouldn't happen, but in v6 we introduced the "quick" button with an inconsistency with other buttons.
+        // the quick button name is converted with `Str::studly()` to get the access string and that makes it impossible
+        // for this function to determine the proper access key. For example, a quick button with the name `comment`
+        // would have the access key `Comment`, while create, update, delete etc have the access keys `create`, `update`, `delete` without transformation.
+        // we should remove the transformation in vendor\backpack\crud\src\resources\views\crud\buttons\quick.blade.php and remove this line from here in v7.
+        $condition = $condition ?? $this->get(Str::camel($operation).'.access');
```

Comment:
Pedro, **what do you think, instead of this long comment... that we keep this as a feature, and don't remove it in v7?** 

Why am I suggesting this?
- Because this is a DX problem that I've encountered too. It's easy to make typo with these, and it's a shitty thing to debug for 30 min. So streamlining the experience here could be a good DX win, as a result of fixing this bug.
- In my opinion, using `hasAccess('inline-edit')` and `hasAccess('InlineEdit')` or `hasAccess('inlineEdit')`... should do the same thing.

Do you agree?